src/js/smooch.jsx

```diff
@@ -190,6 +190,8 @@ export class Smooch extends Observable {
                 this._el = renderWidget();
             }
             this.trigger('ready');
+
+            return store.getState().user;
```

Comment:
Should we also pass this to the `ready` callback? Is it possible? `this.trigger('ready', store.getState().user)`